src/test/java/com/github/paweladamski/httpclientmock/HttpClientMockBuilderTest.java

```diff
@@ -314,6 +319,45 @@ public void should_allow_different_host_then_default() throws IOException {
     assertThat(login, hasContent("login"));
     assertThat(google, hasContent("google"));
   }
+  
```

Comment:
I think there should be also a negative test. I mean we should also test the situation when `UrlEncodedFormCondition` fails.

Reply:
Added in f7d162730ad72a9549c7f2c7afeb45281589eade.